src/query/ts/m3db/encoded_unconsolidated_series_iterator.go

```diff
@@ -85,8 +92,10 @@ func (it *encodedSeriesIterUnconsolidated) Next() bool {
 		return false
 	}
 
-	alignedValues := values.AlignToBoundsNoWriteForward(it.meta.Bounds, it.lookbackDuration)
-	it.series = block.NewUnconsolidatedSeries(alignedValues, it.seriesMeta[it.idx])
+	it.series = block.NewUnconsolidatedSeries(
```

Comment:
Do we not need to `AlignToBoundsNoWriteForward` anymore?

Reply:
No; in fact I've managed to remove it in the cleanup

Reply:
For more context, this was only being used in temporal functions, which immediately unrolled it back into a flat list :P

I've since split consolidated/unconsolidated distinction to be StepIterator = consolidated, SeriesIterator = unconsolidated, as unconsolidated steps or consolidated series don't make a lot of sense